### PR TITLE
WL-3843: For export, get correct variable (contentCollectionId) from page

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by buckett** [comment]
I mentioned in the Jira that this looks to be the wrong issue (from the description), what do you think?

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ protected void handleExportRequest(HttpServletRequest req, HttpServletResponse r
 			Reference ref, String format, String subtype) 
 			throws EntityNotDefinedException, EntityAccessOverloadException, EntityPermissionException 
 	{
-		if(! ContentHostingService.allowGetResource(req.getParameter("resourceId")))
+		if(! ContentHostingService.allowGetResource(req.getParameter("contentCollectionId")))
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java:136]: It seems strange todo a permission check against one request parameter but then do the export based on another parameter.

**Comment by buckett** [comment]
Comments in Jira about this one.

**Comment by nicholaswilson100** [comment]
This issue still isn't fixed after I backed out WL-1001 (and SAK-21748 was already in the 10.x branch), the issue being that 'Export entire list' in the reading list doesn't work/does nothing.  What do you think the best fix for this is?

**Comment by buckett** [comment]
But why is it working on trunk?

**Pull request closed without merging**